### PR TITLE
Bug 1769016: Use crd definition in openshift/api

### PR DESCRIPTION
**Pull request by adambkaplan**
/assign @gabemontero 

/cc @bparees @soltysh 

Looks like I didn't need to bump openshift-controller-manager after all.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -39,7 +39,7 @@ require (
 
 replace (
 	github.com/jteeuwen/go-bindata => github.com/jteeuwen/go-bindata v0.0.0-20151023091102-a0ff2567cfb7
-	github.com/openshift/api => github.com/openshift/api v3.9.1-0.20191028134408-abf28f93703b+incompatible
+	github.com/openshift/api => github.com/openshift/api v3.9.1-0.20191111211345-a27ff30ebf09+incompatible
```
Comment (gabemontero) [inline go.mod]: I've started seeing the pattern in some PRs of removing the replace line once the line in requires matches it (i.e. this line 42 and line 18)

should we jump on this pattern bandwagon ?

**Comment by gabemontero** [comment]
works for me @soltysh @adambkaplan

**Comment by adambkaplan** [comment]
/hold

library-go broke us - needs https://github.com/openshift/library-go/pull/588 and new bump(*) to fix

**Comment by adambkaplan** [comment]
(and @gabemontero I'll fix your nit with the re-vendor)

**Comment by adambkaplan** [comment]
/close

Moved to #132

**Pull request closed without merging**